.jenkins/pytorch/win-test.sh

```diff
@@ -59,3 +62,9 @@ run_tests() {
 }
 
 run_tests && assert_git_not_dirty && echo "TEST PASSED"
+
+if [[ "${BUILD_ENVIRONMENT}" == "pytorch-win-vs2019-cuda10-cudnn7-py3" ]]; then
+  cd $TEST_DIR
```

Comment:
It seems that you'll need to run the following code block in the project root dir.